pkg/user/tx_client.go

```diff
@@ -132,6 +134,20 @@ func WithEstimatorService(conn *grpc.ClientConn) Option {
 	}
 }
 
+// WithAdditionalCoreEndpoints adds additional core endpoints to the TxClient.
+// For transaction submission, the client will attempt to use the primary endpoint
+// and the first two additional endpoints provided via this option.
+func WithAdditionalCoreEndpoints(conns []*grpc.ClientConn) Option {
+	// set 2 additional core endpoints on the `TxClient`
+	if len(conns) > 2 {
+		conns = conns[:2]
+	}
+
+	return func(c *TxClient) {
+		c.conns = append(c.conns, conns...)
```

Comment:
[not blocking] It seems like a developer could call

```go
.WithAdditionalCoreEndpoints([conn1, conn2])
.WithAdditionalCoreEndpoints([conn3, conn4])
```

and get around the 2 endpoint restriction 🤔